views/admin.functions.setup.options.php

```diff
@@ -312,6 +347,11 @@ function sailthru_setup_callback() {
 	echo '<p>Add your Sailthru API key & Secret, you can find this on the <a href="https://my.sailthru.com/settings_api">settings page</a> of the Sailthru dashboard.</p><p>Not sure what these are? Contact <a href="mailto:support@sailthru.com">support@sailthru.com</a> ';
 } // end sailthru_setup_callback
 
+function recaptcha_setup_callback() {
+	echo '<h3 class="sailthru-sub-section">reCaptcha Site Key and Secret</h3>';
+	echo '<p>(Optional) Add reCaptcha to your page to detect potential spamming and bots. <a href="https://www.google.com/recaptcha/intro/v3.html">Visit the reCaptcha homepage</a> to set up a site key and secret.</p>';
```

Comment:
Final wording on this?

Reply:
"Add reCaptcha to your ~~page~~ signup forms"